@navikt/core/react/src/collapsible/parts/Collapsible.Content.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from "react";
+import { Slot } from "../../util/Slot";
+import { useCollapsibleContext } from "../Collapsible.context";
+
+export interface CollapsibleContentProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * When true, will render element as its child. This merges classes, styles and event handlers.
+   * @default false
+   */
+  asChild?: boolean;
+}
+
+export const CollapsibleContent = forwardRef<
+  HTMLDivElement,
+  CollapsibleContentProps
+>(({ children, asChild, ...props }, ref) => {
```

Comment:
Vi pleier å skrive `...rest`
```suggestion
>(({ children, asChild, ...rest }, ref) => {
```